wifi: Add nRF7001DK support to lock sample (#14584)
nrfconnect/sdk-nrf: doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

Matter samples
--------------

> * Added support for nrf7001 DK to Matter Lock Sample.

Reviewer: We don't really have a 7001 DK - better refer to it with its full name (emulation board)

Reviewer 2: Ok, right. Is it better now?

Reviewer: yes, to me at least

> * Support for emulation of nRF7001 Wi-Fi companion IC on nrf7002 DK to Matter Lock Sample.

Reviewer 3: ```suggestion
* :ref:`matter_lock_sample` sample:

  * Added support for emulation of the nRF7001 Wi-Fi companion IC on nRF7002 DK.
```

Matter samples
--------------

* :ref:`matter_lock_sample` sample:

>    * Added support for emulation of the nRF7001 Wi-Fi companion IC on nRF7002 DK.

Reviewer 3: ```suggestion
   * Added support for emulation of the nRF7001 Wi-Fi companion IC on the nRF7002 DK.
```
I missed an article.
